src/cpp/r/R/Api.R

```diff
@@ -303,7 +303,9 @@
    # then use a separate API (this allows the API to work regardless of
    # whether we're in source or visual mode)
    if (identical(line, -1L) && identical(col, -1L))
-      return(invisible(.Call("rs_fileEdit", filePath, PACKAGE = "(embedding)")))
+   {
+      return(invisible(sapply(filePath, .rs.api.documentOpen)))
```

Comment:
`rs_fileEdit` which was used previously is vectorized, i.e. it sends one request per element of `filePath`: 

```cpp
      // fire events
      std::for_each(filenames.begin(), filenames.end(), enqueFileEditEvent);
```

but here when position is supplied, `filePath` is scalar forced: 

```r
    # send event to client
   .rs.enqueClientEvent("jump_to_function", list(
      file_name     = .rs.scalar(filePath),
      line_number   = .rs.scalar(line),
      column_number = .rs.scalar(col),
      move_cursor   = .rs.scalar(moveCursor)))
```

so maybe `filePath` should just be size 1 ?